test/lib/deployment/test-deployment.js

```diff
@@ -29,7 +29,17 @@ async function testDeployment(
   buildDelegate
 ) {
   console.log('testDeployment', fixturePath);
-  const globResult = await glob(`${fixturePath}/**`, { nodir: true });
+
+  // Load nowignore from fixture dir
+  let nowIgnoreGlob = '';
+  const nowIgnoreFile = `${fixturePath}/.nowignore`;
+  if (fs.exists(nowIgnoreFile)) {
+    console.log('nowignore file found', nowIgnoreFile);
+    const ignores = fs.readFileSync(nowIgnoreFile).toString().split('\n').filter((el) => el.length > 0);
+    nowIgnoreGlob = '!(' + ignores.join('|') + ')';
+  }
+
+  const globResult = await glob(`${fixturePath}/**${nowIgnoreGlob}`, { nodir: true });
```

Comment:
Oh, this is cool. Did you consider glob.ignore (https://github.com/isaacs/node-glob#options)?

Reply:
Missed that option. But it makes life easier and more readable. I will change it

Reply:
After trying to change this is is not working as expected. The ignore option expects glob expressions, while in a file like .nowignore there are kind of lazy glob expressions. Take for example the entry `vendor` in such ignore file. The `glob.ignore` option should then be `vendor/**`. Because you can not know if it is a directory or not, the current way of ignoring the files is better in my opinion.

Reply:
I see, thanks.